src/runvm-osbuild

```diff
@@ -114,4 +118,13 @@ osbuild \
 # from the cache qcow2 so we don't cache it.
 mv "${outdir}/${platform}/${filename}" "${filepath}"
 
+# In case of IBM Secure Execution there are more artifacts
+dir=$(dirname "${filepath}")
+if [ -f "${outdir}/${platform}/bootfs_hash" ]; then
+    cp "${outdir}/${platform}/bootfs_hash" "${dir}"
+fi
+if [ -f "${outdir}/${platform}/rootfs_hash" ]; then
+    cp "${outdir}/${platform}/rootfs_hash" "${dir}"
+fi
```

Comment:
Would be cleaner to gate this on `$secex`. And let fail if somehow the hash files don't exist.